features/manage-position/Create.tsx

```diff
@@ -109,12 +115,17 @@ const Create = () => {
   };
 
   const computeCR = () => {
+    if (collateral === null || tokens === null) return null;
+    return parseFloat(collateral) / parseFloat(tokens);
+  };
+  const computedCR = computeCR() || 0;
```

Comment:
Need to distinguish between `computedCR` and `resultingCR`.
- `computedCR`: CR of new tokens to create. This must be > GCR.
- `resultingCR`: CR of position including new tokens to create. This can be below GCR but its useful for user to know their new CR.

Reply:
good catch. Bit of a pain given how this contract is implemented but having both cases helps a bunch.